docs/secure-tikv-cluster.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Secure the TiKV Cluster 
```

Comment:
-> TiKV Security Configuration
(I think readers might expect a step-by-step procedure from the previous title.)